prow/plugins/verify-owners/verify-owners.go

```diff
@@ -126,27 +155,107 @@ func handle(ghc githubClient, gc *git.Client, log *logrus.Entry, pre *github.Pul
 		}
 	}
 
-	// Check each OWNERS file.
+	// If OWNERS_ALIASES file exists, get all aliases.
+	path := filepath.Join(r.Dir, ownersAliasesFileName)
+	if _, err := os.Stat(path); err == nil {
+		b, err := ioutil.ReadFile(path)
+		if err != nil {
+			log.WithError(err).Errorf("Failed to read %s.", path)
+			return nil
+		}
+		repoAliases, err = repoowners.ParseAliasesConfig(b)
+		if err != nil {
+			return fmt.Errorf("error parsing aliases config for %s file: %v", ownersAliasesFileName, err)
+		}
+
+		// If OWNERS_ALIASES file was modified, check if newly added owners are collaborators.
+		if ownerAliasesModified {
+			allOwners := repoAliases.ExpandAllAliases().List()
+			for _, owner := range allOwners {
+				// cap the number of checks to avoid exhausting tokens in case of large OWNERS refactors.
+				if len(nonCollaborators) > 20 {
+					break
+				}
+
+				// look for newly added owners by checking if they are in the patch.
+				if strings.Contains(modifiedOwnerAliasesFile.Patch, owner) {
+					isCollaborator, err := ghc.IsCollaborator(org, repo, owner)
```

Comment:
Would it be more correct to use `trigger.TrustedUser` here?

Reply:
> Would it be more correct to use trigger.TrustedUser here?

Done.